docs/styles.css

```diff
@@ -4,850 +4,458 @@ Fonts
 Note that these are commercially licensed, we have bought a licence for them for use on the site
 but as a result they're not checked into Git.
 */
+/*
```

Comment:
Oops?

Reply:
Not in use. I think I was being cautious to make sure unused assets aren't downloaded.